### PR TITLE
Enable markdown in Sphinx and convert index, readme

### PR DESCRIPTION
**Pull request by catherinedevlin**
Uses m2r to .mdinclude README and CHANGELOG from base directory into docs/changelog.rst and docs/index.rst

This also uses .md files in docs/ to work the same way .rst do.

Closes https://github.com/ZaxR/bulwark/issues/35

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Installation
+============
+
+``` {.sourceCode .bash}
```
Comment (ZaxR) [inline docs/installation.md]: This is being output as:

`{.sourceCode .bash} pip install bulwark`

Screen Shot 2019-07-28 at 11 53 48 AM

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+add_five(df)
+```
+
+You can stack multiple decorators on top of each other to have the first
+failed decorator check result in an assertion error or use the built-in
+MultiCheck to collect all of the errors are raise them at once.
+
+See [examples]{role="ref"} to see more advanced usage.
```
Comment (ZaxR) [inline docs/quickstart.md]: This is outputting as text:

[examples]{role=”ref”}

Comment (catherinedevlin) [reply docs/quickstart.md]: Yup!  Pandoc conversion stumbled here.

File: docs/installation.md (hunk 1 above)
Comment (catherinedevlin) [reply docs/installation.md]: Also a pandoc error

**Comment by catherinedevlin** [comment]
Got these changes made, thanks!

File: docs/conf.py [hunk 3]
```diff
@@ -58,7 +59,7 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ['.rst', ]
```
Comment (ZaxR) [inline docs/conf.py]: Do we want to add .md here? m2r's docs say it should read `['.rst', '.md']`, but sphinx says it should be 
```
source_suffix = {
    '.rst': 'restructuredtext',
    '.md': 'markdown',
}
```

The latter seems to be a more robust syntax, though the docs don't appear to look any different with any of the three options.

**Pull request merged**